decidim/decidim
# Remove designation_mode field from Assembly Members

#### :tophat: What? Why?

While I was writing the [docs for assembly members](https://docs.decidim.org/en/admin/spaces/assemblies/members/), I found out that the `designation_mode` field was unused. This PR removes this field.

#### :pushpin: Related Issues
- Related to #3008

#### Testing

1. Sign in as admin
2. Go to an assembly's member new form 
3. See that this field isn't there anymore

:hearts: Thank you!
